lib/tests/activate-fork

```diff
@@ -0,0 +1,29 @@
+const { expect } = require('chai');
+const rskUtils = require('../rsk-utils');
+const CustomError = require('../CustomError');
+const { getRskTransactionHelpers } = require('../rsk-tx-helper-provider');
+
+const execute = (fork, getFederates) => {
```

Comment:
Maybe out of scope for now, but keep in mind we will only be activating one fork. The upcoming one. So no need to make this too generic

Reply:
Gotcha!